mmv1/products/securitycenterv2/ProjectNotificationConfig.yaml

```diff
@@ -0,0 +1,130 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'ProjectNotificationConfig'
```

Comment:
from debug log output:
```
---[ REQUEST ]---------------------------------------
POST /v2/projects//locations/%7Blocation%7D/notificationConfigs?alt=json&configId=tf-test-my-configp2woyrwrln HTTP/1.1
Host: securitycenter.googleapis.com
User-Agent: Terraform/1.8.3 (+https://www.terraform.io) Terraform-Plugin-SDK/2.33.0 terraform-provider-google-beta/acc
Content-Length: 258
Content-Type: application/json
Accept-Encoding: gzip

{
 "description": "My custom Cloud Security Command Center Finding Notification Configuration",
 "pubsubTopic": "projects/ci-test-project-188019/topics/tf-test-my-topicp2woyrwrln",
 "streamingConfig": {
  "filter": "category = \"OPEN_FIREWALL\" AND state = \"ACTIVE\""
 }
}

-----------------------------------------------------
2024/06/22 01:13:36 [DEBUG] Google API Response Details:
---[ RESPONSE ]--------------------------------------
HTTP/2.0 400 Bad Request
Cache-Control: private
Content-Type: application/json; charset=UTF-8
Date: Sat, 22 Jun 2024 01:13:36 GMT
Server: ESF
Vary: Origin
Vary: X-Origin
Vary: Referer
X-Content-Type-Options: nosniff
X-Frame-Options: SAMEORIGIN
X-Xss-Protection: 0

{
  "error": {
    "code": 400,
    "message": "Request contains an invalid argument.",
    "status": "INVALID_ARGUMENT"
  }
}
```

Reply:
still seeing the same errors:
```hcl
---[ REQUEST ]---------------------------------------
POST /v2/projects//locations/global/notificationConfigs?alt=json&configId=tf-test-my-configswvls8ikvk HTTP/1.1
Host: securitycenter.googleapis.com
User-Agent: Terraform/1.8.3 (+https://www.terraform.io) Terraform-Plugin-SDK/2.33.0 terraform-provider-google-beta/acc
Content-Length: 258
Content-Type: application/json
Accept-Encoding: gzip

{
 "description": "My custom Cloud Security Command Center Finding Notification Configuration",
 "pubsubTopic": "projects/ci-test-project-188019/topics/tf-test-my-topicswvls8ikvk",
 "streamingConfig": {
  "filter": "category = \"OPEN_FIREWALL\" AND state = \"ACTIVE\""
 }
}

-----------------------------------------------------
2024/06/26 21:32:32 [DEBUG] Google API Response Details:
---[ RESPONSE ]--------------------------------------
HTTP/2.0 400 Bad Request
Cache-Control: private
Content-Type: application/json; charset=UTF-8
Date: Wed, 26 Jun 2024 21:32:32 GMT
Server: ESF
Vary: Origin
Vary: X-Origin
Vary: Referer
X-Content-Type-Options: nosniff
X-Frame-Options: SAMEORIGIN
X-Xss-Protection: 0

{
  "error": {
    "code": 400,
    "message": "Request contains an invalid argument.",
    "status": "INVALID_ARGUMENT"
  }
}
```

Reply:
I am confused about using the path param `project` or `project_id` or `projectId`. Because by using `project` it is throwing an error of duplicate mapping and if we use `projectId` it is not showing up. Can you suggest the correct variation to use.

Reply:
Hi vijay, it looks like the best option would be to use `{{project}}`

I see there's some import issues which come from the project param that is being used:
```
2024/07/25 03:57:36 [DEBUG] [transport] [server-transport 0xc00126b800] loopyWriter exiting with error: transport closed by client 
    vcr_utils.go:152: Step 2/2 error running import: ImportStateVerify attributes not equivalent. Difference is shown below. The - symbol indicates attributes missing after import.
        
          map[string]string{
        - 	"project": "ci-test-project-188019",
        + 	"project": "1067888929963",
          }
```

could you provide the output you get when attempting to use`project`?